### PR TITLE
Desktop: Support scrolling in the note list using keys (eg page up, page down)

### PR DESCRIPTION
**Pull request by mic704b**
Please consider this PR to implement navigation in the note list using standard keys.

# Description
Currently using navigation keys in the note list does not work after the first key. This PR implements these functions for standard keys (Mac, Windows, Linux):

**Navigation**  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;**Key**

Up one line  &nbsp; &nbsp;&nbsp;&nbsp; &nbsp; `Up`&nbsp;&nbsp;&nbsp; (already supported)
Down one line  &nbsp;&nbsp; `Down`&nbsp;&nbsp;&nbsp; (already supported)
Up one page &nbsp;&nbsp;&nbsp;&nbsp; `Page Up`
Down one page `Page Down`
Start of list  &nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `Cmd+Up`, `Ctrl+Home`
End of list   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp; `Cmd+Down`, `Ctrl+End`

# Tests
Manual testing in note list:
- [x] Line up / down
- [x] Page up / down
- [x] Start / end

# Discussion
If the user has scrolled the selected item off the screen, then none of the key actions work. This issue is pre-existing and not related to any change in this PR.

### CODE REVIEW

**Comment by tessus** [comment]
This could also be useful for the folder list.

File: ElectronClient/app/gui/NoteList.jsx [hunk 1]
```diff
@@ -293,20 +293,43 @@ class NoteListComponent extends React.Component {
...
+		}
+		if ((keyCode === 36 && ctrlKey) || (keyCode === 38 && metaKey)) {
+			noteIndex = 0;
+		}
+		if (keyCode === 38 && !metaKey) {
+			noteIndex -= 1;
+		}
+		if (keyCode === 40 && !metaKey) {
+			noteIndex += 1;
+		}
```
Comment (laurent22) [inline ElectronClient/app/gui/NoteList.jsx]: Please add the name of the key in a comment, next to the keycode as this make it easier to see what is what.

Comment (mic704b) [reply ElectronClient/app/gui/NoteList.jsx]: Done

**Comment by laurent22** [comment]
Perfect, thanks!

**Pull request merged**